config/gadi.config

```diff
@@ -1,38 +1,93 @@
 // profile for the Gadi supercomputer at NCI
-// Before running will need to run: 
-// module load nextflow singularity 
-
-// Gadi specific parameters
 // More info: https://opus.nci.org.au/display/DAE/Nextflow
-params {
-    pbs_account = ''
-    whoami = ''
-}
 
 // Preset use of containers with Singularity
 singularity {
     enabled = true
-    cacheDir = "/scratch/${params.pbs_account}/${params.whoami}/.nextflow/singularity"
+    cacheDir = "/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/singularity_cache"
     autoMounts = true
-    temp = "scratch/${params.pbs_account}/${params.whoami}/containers"
+    autoCleanUp = true
 }
 
 process {
-    module = 'singularity'
-    cache = 'lenient'
-    stageInMode = 'symlink'
-    executor = 'pbspro'
-    project = ''
-    storage = 'scratch/'
-    queue = 'normal'
-    
-    withName: 'processOne' {
+	//beforeScript = '${projectDir}/scripts/pullContainers.sh'
+	module = 'singularity'
+	cache = 'lenient'
+	stageInMode = 'symlink'
+	project = "${params.gadi_account}"
+	storage = "scratch/${params.gadi_account}+gdata/${params.gadi_account}"
+	disk = '30.GB'
+        
+
+withName: 'checkInputs' {
+	executor = 'local'
+}
+
+
+withName: 'mutect2' {
+        executor = 'pbspro'
```

Comment:
Remember to follow the [DRY principles](https://www.baeldung.com/cs/dry-software-design-principle) when writing code. If all processes are run as pbs jobs, can set executor at the same level as `withName`, inside `process{}`, rather than for each individual process.

Reply:
This is still unresolved and needs to be addressed.